static/app/views/issueList/overviewFc.tsx

```diff
@@ -355,7 +370,6 @@ function IssueListOverviewFc({location, router}: Props) {
 
   const trackTabViewed = useCallback(
     (newGroupIds: string[], data: Group[], numHits: number | null) => {
-      const page = location.query.page;
```

Comment:
Not a fan of `useLocationQuery`? It's pretty handy for this kind of decoding, and setting defaults!

Reply:
Good callout! This component just got converted to an FC recently so that's something I'll add for sure